### ISSUE TITLE
Support ignoring of SSL exceptions on Android.

### ISSUE THREAD

**Issue by RobertZenz** [floccusaddon/floccus#1435]
### Describe the feature you'd like to request

The ability to ignore SSL exceptions and errors in the Android app when connecting to a server.

### Describe the solution you'd like

Currently when trying to connect to a machine in the local network, for example "https://192.168.0.128/" an error message is displayed that an SSL-Exception was thrown. That is to be expected, as "192.168.0.128" will never match the certificate. Being able to ignore that exception would be great.

Basically what is needed is the possibility to accept self-signed certificates on Android and ignore when there is a problem with the certificate (TLD of certificate not matching address).

### Describe alternatives you've considered

The most suggested "solution" is to have your router "reroute" the TLD provided in the certificate of the machine to the local machine within the local network. In many setups this is neither feasible nor possible.

**Comment by marcelklehr** [comment]
Hello @RobertZenz 

due to security concerns I will not add a setting to floccus that allows people to ignore TLS errors and warnings. Self-signed certificates have known drawbacks that anyone will need to mitigate themselves if they choose to go down that path. Getting android and android apps to accept them is one of these drawbacks. See #208 for discussions about this. 
In my opinion there is a tested and true path even for selfhosting at home: Get a cheap domain and a free certificate from letsencrypt, or sign up for a subdomain on a service like duckdns.
Please understand that I need to close this request.
All the best for your future endeavors!
